src/sentry/monitors/clock_dispatch.py

```diff
@@ -85,7 +86,12 @@ def _dispatch_tick(ts: datetime):
         # XXX(epurkhiser): Unclear what we want to do if we're not using kafka
         return
 
-    message: ClockTick = {"ts": ts.timestamp()}
+    volume_anomaly_result = _safe_evaluate_tick_decision(ts)
```

Comment:
Moved tick evaluation into the clock dispatch, which probably makes more sense to be the thing to call it